src/bsconfig.rs

```diff
@@ -179,6 +179,9 @@ pub struct T {
     // this is a new feature of rewatch, and it's not part of the bsconfig.json spec
     #[serde(rename = "namespace-entry")]
     pub namespace_entry: Option<String>,
+    // this is a new feature of rewatch, and it's not part of the bsconfig.json spec
+    #[serde(rename = "allowed-parents")]
+    pub allowed_parents: Option<Vec<String>>,
```

Comment:
`allowed-dependents` (I think dependents is a better term for the opposite of dependencies)